src/lib/analytics.ts

```diff
@@ -26,13 +24,12 @@ export const identify = (property: AnalyticsUserProperty) => {
 };
 
 export const track = (event: AnalyticsEvent) => {
-  const eventDataWithReferrer = { ...(event.payload ?? {}), referrer: testFlags.referrer };
```

Comment:
why is this removed?

Reply:
oh because referrer is automatically tracked by amplitude?

Reply:
yup! utm_source is already tracked by amplitude and i have a feeling it's actually slightly more accurate, so i'm just getting rid of it to reduce confusion